### PR TITLE
[Ready to merge after Doc Freeze] Document Cloud metrics with multiple dimension combinations

### PR DESCRIPTION
**Pull request by morecar**
Document metrics with multiple dimension combinations:
* Clarification on metric definition including both metric name and dimensions 
* Legacy by+dimension
* How to filter out dimensions that are not desired

### CODE REVIEW

**Review by mkrichevsky — APPROVED** [review]

**Review by mkrichevsky — APPROVED** [review]

**Review by jeff-colucci — APPROVED** [review]
Fixed the tables and some styling, looks good other than that!

**Review by jeff-colucci — APPROVED** [review]

**Comment by morecar** [comment]
I fixed the conflict and a double ## in one of the sections.

File: src/content/docs/infrastructure/amazon-integrations/get-started/aws-integrations-metrics.mdx [hunk 1]
```diff
@@ -28,27 +29,35 @@ Metrics received from AWS CloudWatch are stored in New Relic as [dimensional met
...
-      </th>
-    </tr>
+### Metrics with the `by + dimension ` suffix
+
+For historical reasons, we suffic some metrics that are part of pairs where one of the metric has a dimension but the other doesn't. In these cases we use the `by + dimension` suffix on the metric that has the dimension. For example: 
+
+* `aws.lambda.Duration` with dimensions `aws.lambda.FunctionName` is mapped to `aws.lambda.Duration.byFunction`
+* `aws.lambda.Duration` with no dimensions stays the same. 
+
+We are no longer adding these mappings, and we're are commited to preserve all the new metrics with their original AWS name, as defined by the general rule above. The following is a complete list of all the metrics that are suffixed with `.by + dimensionName`:
```
[the remark this answers is not in the document]
Comment (morecar) [reply src/content/docs/infrastructure/amazon-integrations/get-started/aws-integrations-metrics.mdx:53]: We are no longer creating these mappings based on metric name + dimensions, the ones that are in the list are the ones that will ever exist.

File: src/content/docs/infrastructure/amazon-integrations/get-started/aws-integrations-metrics.mdx [hunk 2]
```diff
@@ -28,27 +29,35 @@ Metrics received from AWS CloudWatch are stored in New Relic as [dimensional met
...
+* `aws.apigateway.4XXError` with dimensions `aws.apigateway.ApiName` and `aws.apigateway.Stage`.
+* `aws.apigateway.4XXError` with dimensions `aws.apigateway.ApiName`.
 
-<table>
-  <tbody>
-    <tr>
-      <th style={{ width: "100px" }}>
-        Integration
-      </th>
+Each of these metrics is ingested as an invididual time series.
```
[the remark this answers is not in the document]
Comment (morecar) [reply src/content/docs/infrastructure/amazon-integrations/get-started/aws-integrations-metrics.mdx:40]: Added some NRQL examples to clarify

[the remark this answers is not in the document]
Comment (morecar) [reply src/content/docs/infrastructure/amazon-integrations/get-started/aws-integrations-metrics.mdx:40]: Both AWS and Azure examples

**Review by jeff-colucci — APPROVED** [review]
Looks good to me!

**Review by jeff-colucci — DISMISSED** [review]

**Review by jeff-colucci — APPROVED** [review]

**Pull request merged**